locales/en/fields.json

```diff
@@ -70,6 +70,9 @@
       "prison_recall": {
         "label": "Prison recall"
       },
+      "police_transfer": {
+        "label": "Police custody (In force moves only)"
```

Comment:
Should this include a hyphen `In-force`? Or better yet...do we even need to say this seeing as we have only enabled for this Police users? Ie, the beauty of digital vs paper...

Reply:
`In force` refers to the individual constabularies not the Police as a whole. And apparently, no, no hyphen.